src/cli.rs

```diff
@@ -168,8 +168,8 @@ Example: --print=Hb"
     pub quiet: bool,
 
     /// Always stream the response body.
-    #[clap(short = 'S', long)]
-    pub stream: bool,
+    #[clap(short = 'S', long, default_missing_value = "true", num_args= 0..=1, require_equals = true)]
```

Comment:
Is there a reason to prefer `Option<bool>` over `bool`? It affects the generated help output, but in terms of functionality, the result is the same: `false`/`true` versus `None`/`Some(true)`

#### Before
```
-S, --stream      Always stream the response body
```
#### After
```
-S, --stream[=<STREAM>]   Always stream the response body [possible values: true, false]
```

Reply:
I think we need to provide users with the ability to choose to turn off automatic print body as stream.

|cli args|stream value|description|
|--|--|--|
xh -S| Some(true) | alway enable stream  |
xh -S=true| Some(true) |  alway enable stream   |
xh -S=false|  Some(false) |  alway disable stream 
xh|None| auto enable stream when meet stream body |